### PR TITLE
[Excel] (Chart) Update chart activated sample

### PR DESCRIPTION
**Pull request by alison-mk**
Addresses [customer issue #980 in office-js-docs-reference](https://github.com/OfficeDev/office-js-docs-reference/issues/980).

### CODE REVIEW

File: samples/excel/30-events/events-chart-activated.yaml [hunk 1]
```diff
@@ -28,7 +29,16 @@ script:
 
         async function chartActivated(event) {
             await Excel.run(async (context) => {
-                console.log("The pie chart is the active chart. ID: " + event.chartId);
+                // Retrieve the worksheet.
+                var sheet = context.workbook.worksheets.getActiveWorksheet();
```
Comment (AlexJerabek) [inline samples/excel/30-events/events-chart-activated.yaml:33]: The samples favor `const` and `let` over `var`.
```suggestion
                const sheet = context.workbook.worksheets.getActiveWorksheet();
```

File: samples/excel/30-events/events-chart-activated.yaml [hunk 2]
```diff
@@ -28,7 +29,16 @@ script:
...
+
+                // Retrieve the activated chart by ID and load the name of the chart. 
+                var activatedChart = sheet.charts.getItem(event.chartId);
```
Comment (AlexJerabek) [inline samples/excel/30-events/events-chart-activated.yaml:36]:
```suggestion
                const activatedChart = sheet.charts.getItem(event.chartId);
```

**Review by AlexJerabek — APPROVED** [review]
Looks good. Should we also change the "Chart collection events" sample to match? That also has an `onActivated` event.

**Comment by alison-mk** [comment]
@AlexJerabek Good idea! I've tracked this suggestion as a [work item](https://office.visualstudio.com/OC/_workitems/edit/5199402).

**Pull request merged**